ansible-deployment/ovs-interface/tasks/main.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Copy network scripts and bring up interfaces.
```

Comment:
is it not possible to use the `linux-system-roles.network` role like you are using for bringing up other interfaces? Only curious, i dont know the right answer.

Reply:
Yes. I can do it in the same way as other interfaces.